src/ontology/mondo-ingest.Makefile

```diff
@@ -224,13 +224,23 @@ $(REPORTDIR)/%_excluded_terms_in_mondo_xrefs.tsv $(REPORTDIR)/%_excluded_terms_i
 	--component-signature-path $(REPORTDIR)/component_signature-$*.tsv \
 	--outpath $@
 
+# Exclusions: specific ontologies
+.PHONY: omim-intensional-exclusions-add-non-diseases
+omim-intensional-exclusions-add-non-diseases:
+	wget https://github.com/monarch-initiative/omim/releases/latest/download/mondo_non_disease_exclusions.tsv -O config/omim_non_disease_exclusions.tsv
+	python3 $(SCRIPTSDIR)/omim_intensional_exclusions_add_non_diseases.py \
+	  --intensional-exclusions-path config/omim_exclusions.tsv \
+	  --non-disease-exclusions-path config/omim_non_disease_exclusions.tsv
+	rm config/omim_non_disease_exclusions.tsv
+
+
 # Exclusions: all artefacts for single ontology
 exclusions-%:
 	$(MAKE) $(REPORTDIR)/$*_term_exclusions.txt
 	$(MAKE) $(REPORTDIR)/$*_exclusion_reasons.ttl
 	$(MAKE) $(REPORTDIR)/$*_excluded_terms_in_mondo_xrefs.tsv
 
-exclusions-all:
+exclusions-all: omim-intensional-exclusions-add-non-diseases
```

Comment:
@matentzn I made `omim-intensional-exclusions-add-non-diseases` a dependency of `exclusions-all`. However, I did not add it as a dependency to `$(REPORTDIR)/%_exclusion_reasons.robot.template.tsv`, since that is not OMIM-specific. This means that running OMIM-related exclusions will sometimes update with the latest non-diseases. I'm not really sure how best do do this given our usual design.